examples/transportation/docusaurus/turn-restriction-01-source.yaml

```diff
@@ -18,8 +18,9 @@ properties:
     - overture:transportation:example:simple-turn-restriction-connector2
   road:
     restrictions:
-      turns:
-        - segmentId: overture:transportation:example:simple-turn-restriction-target
-          connectorId: overture:transportation:example:simple-turn-restriction-connector2
-          direction: forward
-          toDirection: forward
+      prohibitedTransitions:
+         - sequence:  
+           - segmentId: overture:transportation:example:simple-turn-restriction-target
+             connectorId: overture:transportation:example:simple-turn-restriction-connector2
+           finalHeading: forward
+           when: {heading : forward} 
```

Comment:
Nit but the space before the colon (` :`) should be deleted to keep roughly the same style throughout.

(Also it will make it consistent with the below `turn-restriction-02-source.yaml` at line 38.)

Reply:
Agree

Reply:
Fixed